thinknimble/tn-spa-bootstrapper
# Add jinja2-time

## What this does

Adds `jinja2-time` library to fix `Unable to load extension: No module named 'jinja2_time'` error when running cookiecutter to generate a new project.

## How to test

1. Create a new project
2. Verify that error does not show and cookiecutter prompt appears
